src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=root
+spring.datasource.password=
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
```

Comment:
what does this property do? Never seen it used before.

Reply:
I think spring boot has added it automaticly when i chose H2 dependencies, it probably change native sql to h2 sql.